portfolio/src/main/webapp/gallery.html

```diff
@@ -17,17 +17,17 @@ <h1>My Image Gallary</h1>
           <img class="pic" src="images/cat2.jpg" alt="My cat and his new bed">
           <img class="pic" src="images/cat3.jpg" alt="My cat taking over my parents' bed">
           <img class="pic" src="images/cat4.jpg" alt="My cat on his first birthday">
-          <button class="s-btn" onclick="flip(-1, 1)"><</button>
-          <button class="btn-right btn" onclick="flip(1, 1)">></button>
+          <button class="slide-btn" onclick="flip(-1, 1)"><</button>
```

Comment:
I only changed style in this PR. Will refactor methods into controller in another PR.

Reply:
Put notes like these in the PR description. :)

Reply:
get it :)